spec/unit/jobs/runtime/expired_blob_cleanup_spec.rb

```diff
@@ -13,21 +13,14 @@ module Jobs::Runtime
 
       describe 'droplets' do
         context 'expired' do
-          let!(:expired_droplet) { DropletModel.make(state: DropletModel::EXPIRED_STATE) }
```

Comment:
I think this file does not need to be changed (as also `expired_blob_cleanup.rb` did not change). It checks that the `exclude(droplet_hash: nil)` is working.

Reply:
Yes, but we want to emphasise that there are two types of droplets: For buildpack apps and for Docker apps. The `expired_blob_cleanup.rb` coding is ok. It correctly filters droplet db entries with `droplet_hash: nil`. This enhancement of the test prevents regressions if someone changes the filtering.

Reply:
But there were already two tests for that:

1. [it 'enqueues a deletion job when droplet_hash is not nil'](https://github.com/cloudfoundry/cloud_controller_ng/blob/a63cfe80d92e8f153db89d10963f10f9173a2f69/spec/unit/jobs/runtime/expired_blob_cleanup_spec.rb#L19-L20)

2. [it 'does nothing when droplet_hash is nil'](https://github.com/cloudfoundry/cloud_controller_ng/blob/a63cfe80d92e8f153db89d10963f10f9173a2f69/spec/unit/jobs/runtime/expired_blob_cleanup_spec.rb#L26-L27)

Reply:
Yes, but there was no "Docker"-related test object in this spec. At first, droplets were only used for CF buildpack apps. Then later the Docker feature was added. The droplet concept was reused, but with a different attribute configuration (`droplet_hash: nil` and instead a `docker_receipt_image`). We wanted to express this fact more clearly in the spec by adding one Docker droplet test object.

Reply:
Okay, now I understand your intention.

Then I suggest to put the "Docker" droplet into a `let!` and give it a descriptive name (maybe `expired_docker_droplet`). And to distinguish between these two types of droplets, I would also suggest to rename the `expired_droplet` (maybe to `expired_buildpack_droplet`?).

And as you focus more on the semantics (and not the technical details of `droplet_hash` being `nil` or not) the spec could be changed to `it 'enqueues a deletion job for the buildpack droplet only'`.

There is also another context below (`context 'failed'`) with a similar intention. This one should be changed in a similar way to be consistent.